calico/election.py

```diff
@@ -175,7 +175,7 @@ def _become_master(self):
             self._etcd_client.write(self._key,
                                     id_string,
                                     ttl=self._ttl,
-                                    prevExists=False,
+                                    prevExist=False,
```

Comment:
No harm in being defensive but have you seen any issues with this fix in place?  It's certainly incorrect without this, python-etcd just swallows the incorrect argument and each worker clobbers the value :-(

I wrote a little repro script that hammers on etcd (2.0.0) with 100 worker processes (yay `multiprocessing`) and I can't seem to hit the window you mention.  It was all simple python, no eventlet and no Neutron forking.  If we do hit issues in Neutron but not in my test, it may be an artefact of the fork or the eventlet monkey-patch instead.